packages/flutter_tools/lib/src/commands/assemble.dart

```diff
@@ -156,7 +156,7 @@ class AssembleCommand extends FlutterCommand {
     ));
     if (!result.success) {
       for (MapEntry<String, ExceptionMeasurement> data in result.exceptions.entries) {
-        printError('Target ${data.key} failed: ${data.value.exception}', stackTrace: data.value.stackTrace);
+        printError('Target ${data.key} failed: ${data.value.exception}');
```

Comment:
Two comments:
- I'm seeing a few other potentially similar cases in aot.dart, bundle.dart, and web/compile.dart.
- Is there any way that an `ExceptionMeasurement` can indicate whether the stack trace should be printed?